myfitnesspal/entry.py

```diff
@@ -1,11 +1,22 @@
 from myfitnesspal.base import MFPBase
+import re
 
 
 class Entry(MFPBase):
     def __init__(self, name, nutrition):
         self._name = name
         self._nutrition = nutrition
 
+
+    #split out quantity and measuring unit out of entry name
```

Comment:
Please indent this by a notch (should be at the same indentation level as `regex` below.  If it already looks like it's at the same level, make sure you're using spaces for indentation (per PEP-8).